assets/models/system/mistralai-Mixtral-8x7B-Instruct-v01/model.yaml

```diff
@@ -1,6 +1,6 @@
 path:
   container_name: models
-  container_path: huggingface/mistralai-Mixtral-8x7B-Instruct-v0-1/1704390002/mlflow_model_folder
+  container_path: huggingface/mistralai-Mixtral-8x7B-Instruct-v0-1/20240306/mistralai-Mixtral-8x7B-Instruct-v0-1/mlflow_model_folder
```

Comment:
this sub folder does not seem to be necessary

Reply:
will not add the second folder next time model release